### PR TITLE
Sync crds to Helm chart

### LINKED ISSUE #240: [Feature] Add the CRD (RayCluster) into the kuberay-operator helm chart
### Search before asking

- [X] I had searched in the [issues](https://github.com/ray-project/kuberay/issues) and found no similar feature requirement.

### Description

I think we should add the CRD (RayCluster) into the Helm chart, so that the users can install the operator only by the chart, no other CRD not found errors.

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by haoxins**
## Related issue number

Closes #240 

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/Makefile [hunk 1]
```diff
@@ -41,6 +41,7 @@ help: ## Display this help.
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=kuberay-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(KUSTOMIZE) build config/crd > ../helm-chart/kuberay-operator/crds/crd.yaml
```
Comment (brucez-anyscale) [inline ray-operator/Makefile:44]: do we want to put rbac also?

**Comment by Jeffwan** [comment]
Thanks for the change. I am thinking whether we should have a separate make command just for sync instead of making changes to existing commands generated from kubebuilder. 

WDYT?

**Comment by haoxins** [comment]
>  I am thinking whether we should have a separate make command just for sync instead of making changes to existing commands generated from kubebuilder.

Both are fine to me. I used this solution in my personal project.
What is the command name you guys prefer?

**Comment by haoxins** [comment]
> 

Added `make helm`

**Comment by Jeffwan** [comment]
Seems the failed test is not related to the change. Let me restart it and see what happened

**Pull request merged**